apps/meteor/app/authentication/server/lib/restrictLoginAttempts.ts

```diff
@@ -121,9 +130,16 @@ export const isValidAttemptByUser = async (login: ILoginAttempt): Promise<boolea
 		return true;
 	}
 
+	const lastBlockAt = (await ServerEvents.findLastBlockByUsername(user.username as string))?.ts;
+	const failedAttemptsSinceLastBlock = await ServerEvents.countFailedAttemptsByUsernameSince(user.username, new Date(lastBlockAt));
+	if (lastBlockAt && lastBlockAt < new Date() && attemptsUntilBlock && failedAttemptsSinceLastBlock < attemptsUntilBlock) {
```

Comment:
`lastBlockAt < new Date()`

Please, correct me if I'm wrong, but isn't this going to be ALWAYS true, since the lastBlockAt is also a `Date()` that cannot be in the future? a `new Date()` would always be bigger than a Date set before...